### PR TITLE
Dropdown Component

### LINKED ISSUE #84: Dropdown component
As an author, I want to have the possibility to add a [Dropdown](https://bulma.io/documentation/components/dropdown/) component.

Additional details:
- please see a dropdown item from the navbar

### PR DESCRIPTION
**Pull request by gfazek**
## Description
Implementing bulma drop down component. Dropdown items are limited to be links for now, enhancement is considerable in the future.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) labeled with `bug`
- [x] New feature (non-breaking change which adds functionality) labeled with `enhancement`
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code standards](../CONTRIBUTING.md) of this project.
- [ ] My change requires updating the documentation. I have updated the documentation accordingly.

### CODE REVIEW

File: application/backend/src/main/java/pl/ds/bulma/components/models/DropdownElementComponent.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pl.ds.bulma.components.models;
+
+import javax.inject.Inject;
+import lombok.Getter;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.models.annotations.Default;
```
Comment (paweljozwicki) [inline application/backend/src/main/java/pl/ds/bulma/components/models/DropdownElementComponent.java]: This import looks like unused. Can we remove it?

File: application/backend/src/test/java/pl/ds/bulma/components/models/DropdownComponentTest.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pl.ds.bulma.components.models;
+
+import static java.util.Objects.requireNonNull;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.function.Predicate;
```
Comment (paweljozwicki) [inline application/backend/src/test/java/pl/ds/bulma/components/models/DropdownComponentTest.java]: This one as well looks like unused.

File: application/backend/src/main/java/pl/ds/bulma/components/models/DropdownElementComponent.java (hunk 1 above)
Comment (gfazek) [reply application/backend/src/main/java/pl/ds/bulma/components/models/DropdownElementComponent.java]: Removed

File: application/backend/src/test/java/pl/ds/bulma/components/models/DropdownComponentTest.java (hunk 2 above)
Comment (gfazek) [reply application/backend/src/test/java/pl/ds/bulma/components/models/DropdownComponentTest.java]: Removed

**Comment by gfazek** [comment]
Added dropdown client side (js) logic.

**Review by paweljozwicki — APPROVED** [review]

**Pull request merged**